update ui label (#9585)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/manage-access/scim-manage-user-licenses.md

## Enable SCIM license mapping

> To use license management using SCIM, go to your **Account settings** > **SSO & SCIM**. Under the **SCIM** section, enable **Ignore dbt license mapping?**. This setting enforces license type for a user based on their SCIM attribute and disable the license mapping and manual configuration set up in dbt.

Reviewer: Same comment as [here](https://github.com/dbt-labs/docs.getdbt.com/pull/9585#discussion_r3558328495)

> To use license management using SCIM, go to your **Account settings** > **SSO & SCIM**. Under the **SCIM** section, enable **Ignore dbt license mapping?**. This setting enforces license type for a user based on their SCIM attribute and disable the license mapping and manual configuration set up in dbt.

Reviewer: ```suggestion
To manage user licenses with SCIM, go to **Account settings** > **SSO & SCIM**. Under the **SCIM** section, enable the **Ignore dbt license mapping** toggle. This setting enforces license type for a user based on their SCIM attribute and disables the license mapping and manual configuration set up in dbt.
```
